src/viam/ur/module/ur_arm.cpp

```diff
@@ -68,6 +68,24 @@ T find_config_attribute(const ResourceConfig& cfg, const std::string& attribute)
 
 enum class TrajectoryStatus { k_running = 1, k_cancelled = 2, k_stopped = 3 };
 
+template <typename Callable>
```

Comment:
Newer versions of boost have this, but not the one in our docker image, so I've rolled my own kinda "meh" one, but good enough.

This is basically how you write "finally" in C++.